server/src/main/java/io/druid/segment/realtime/plumber/RealtimePlumber.java

```diff
@@ -130,6 +132,17 @@
   private static final String COMMIT_METADATA_KEY = "%commitMetadata%";
   private static final String COMMIT_METADATA_TIMESTAMP_KEY = "%commitMetadataTimestamp%";
 
+  private static final ThreadMXBean THREAD_MX_BEAN = ManagementFactory.getThreadMXBean();
+
+  private static long safeGetThreadCpuTime()
```

Comment:
I think I've seen that code elsewhere before, we should probably consolidate it in a common place for everything that needs cpu time?